src/main/java/com/faforever/client/fx/contextmenu/CopyUsernameMenuItem.java

```diff
@@ -0,0 +1,34 @@
+package com.faforever.client.fx.contextmenu;
+
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.util.ClipboardUtil;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Primary
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+@RequiredArgsConstructor
+public class CopyUsernameMenuItem extends AbstractMenuItem<String> {
```

Comment:
Plese see https://github.com/FAForever/downlords-faf-client/pull/2513#discussion_r782447688

Reply:
At the moment I don't think any labels change but it would be good to extend it so that it can in the future in case some server message triggers things. Also actually in case this is used for things like the game title or the map name in games which can be changed by the user.